backend/geonature/migrations/versions/8888e5cce63b_add_index_on_id_area_for_table_cor_area_.py

```diff
@@ -0,0 +1,31 @@
+"""add index on id_area for table cor_area_synthese
+
+Revision ID: 8888e5cce63b
+Revises: 09a637f06b96
+Create Date: 2023-01-18 17:34:54.298323
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "8888e5cce63b"
+down_revision = "09a637f06b96"
+branch_labels = None
+depends_on = None
+
+SCHEMA_NAME = "gn_synthese"
+TABLE_NAME = "cor_area_synthese"
+INDEX_NAME = "index__cor_area__synthese_id_area"
```

Comment:
```suggestion
INDEX_NAME = "i_cor_area_synthese_id_area"
```
Pour suivre les pratiques existantes.